app/models/InputConfiguration.ts

```diff
@@ -1,5 +1,5 @@
 export type InputConfiguration = {
     type: string;
     name: string;
-    storageRelativePath: string;
+    enabled: boolean;
```

Comment:
The `enabled` field is fine. However, don't delete the `storageRelativePath` field.